openpype/pipeline/create/context.py

```diff
@@ -1458,6 +1458,65 @@ def create(self, identifier, *args, **kwargs):
                     identifier, label, exc_info, add_traceback
                 )
             ])
+        return result
+
+    def create(
+        self,
+        creator_identifier,
+        variant=None,
+        asset_doc=None,
+        task_name=None,
+        pre_create_data=None
+    ):
+        """Trigger create of plugins with standartized
+
+        Args:
+            creator_identifier (str):
+            variant (str): Variant used for subset name.
+            asset_doc (Dict[str, Any]):
+            task_name (str): Name of task to which is context related.
+            pre_create_data (Dict[str, Any]): Pre-create attribute values.
+
+        Returns:
+            Any: Output of triggered creator's 'create' method.
+
+        Raises:
+            CreatorsCreateFailed: When creation fails.
+        """
+
+        if pre_create_data is None:
+            pre_create_data = {}
+
+        project_name = self.project_name
+        if asset_doc is None:
+            asset_name = self.get_current_asset_name()
+            asset_doc = get_asset_by_name(project_name, asset_name)
+            task_name = self.get_current_task_name()
+
+        creator = self.creators.get(creator_identifier)
```

Comment:
I feel like the check whether the Creator exists based on the identifier might need to be the first thing we check for? Everything else we're doing doesn't make any sense if we don't know what to create to begin with. It looks very odd that we're trying to continue with logic here even if the creator can't be identified.